src/evlib/processing/reconstruction/e2vid/utils/timers.py

```diff
@@ -0,0 +1,57 @@
+import torch
```

Comment:
In general, I like to delete codes if possible. Can you delete this entire file?